web/modules/custom/dpl_webform/dpl_webform.install

```diff
@@ -107,3 +108,89 @@ function dpl_webform_update_10004(): string {
 
   return implode("\n", $feedback);
 }
+
```

Comment:
You have a typo in the commit msh:
```

from the ignore list. We don't want these config ignored, as it prvent us

Reply:
Fixed now. But I might update commit message after updating the PR.